gadget-container/Makefile

```diff
@@ -53,9 +53,9 @@ push:
 minikube: docker-gadget/minikube-build docker-gadget/minikube-install
 
 docker-gadget/minikube-build:
-	cp ../../traceloop/traceloop ./
+	cp ../../traceloop/traceloop ./bin/
```

Comment:
I think you should add `mkdir -p bin` before, like it's done in the other targets, in case this target is executed independently of the others.